client/src/components/Overview/Overview.jsx

```diff
@@ -32,20 +32,20 @@ const Overview = ({productId}) => {
   // }, [currentStyle]);
 
   return (
-    <div>
+    <div style={{display: 'flex-wrap'}}>
       <div> <ImageGallery
         currentStyle={currentStyle}/> </div>
       <div> <ProductInformation
         productId={productId}
         currentStyle = {currentStyle}/>
       </div>
-      <div> <StyleSelector
+      {productStyles ? <div> <StyleSelector
```

Comment:
i don't think you necessarily need the <div> </div> around <StyleSelector /> here because the StyleSelector component already returns a JSX element wrapped in <div></div>